janusgraph-core/src/main/java/org/janusgraph/core/schema/SchemaManager.java

```diff
@@ -54,5 +58,33 @@
      */
     VertexLabelMaker makeVertexLabel(String name);
 
+    /**
+     * Allows define mapping constraints for a vertex which properties are allowed.
```

Comment:
The whole sentence doesn't look right. How about _Add constraints on which properties are allowed for a given vertex label._?